src/libclipper/include/clipper/containers.hpp

```diff
@@ -5,14 +5,15 @@
 #include <memory>
 #include <unordered_map>
 
+#include <boost/circular_buffer.hpp>
 #include <boost/thread.hpp>
 
 #include <clipper/datatypes.hpp>
 #include <clipper/util.hpp>
 
 namespace clipper {
 
-using Deadline = std::chrono::time_point<std::chrono::high_resolution_clock>;
+using Deadline = std::chrono::time_point<std::chrono::system_clock>;
 
```

Comment:
Sounds good. Not super urgent but using a consistent clock is definitely a good thing in the long run. Can you add a comment explaining why we use system clock here?